core/include/keyman/keyman_core_api.h

```diff
@@ -242,6 +242,12 @@ typedef struct {
 
   // -1=unchanged, 0=off, 1=on
   km_core_caps_state new_caps_lock_state;
+
+  // reference copy of actual UTF32 codepoints deleted from rhs of context
```

Comment:
I think in the past we avoided "rhs" in the past due to right-to-left orientation. I believe kmx_context refers to the caret.
```reference copy of actual UTF32 codepoints deleted from the end of the context closest to the caret```